src/modules/login/Login.scss

```diff
@@ -4,6 +4,7 @@
   @include centered;
   flex-flow: column;
   min-height: 100vh;
+  padding: 80px 0;
```

Comment:
Is this padding same as in Register page ? It should be 60px

Reply:
![image](https://user-images.githubusercontent.com/35345462/97866879-12ac6380-1d0d-11eb-87ec-ea0da9443ff1.png)

It is 80px.